src/sass/formio.form.builder.scss

```diff
@@ -168,3 +168,7 @@
 .help-block {
   margin: 0px;
 }
+
+.btn {
```

Comment:
Can we make this more specific?.... this should only affect buttons in the sidebar.

Reply:
easy fix... I will fix it.